recipe/meta.yaml

```diff
@@ -24,21 +24,15 @@ requirements:
     - pandas
     - giddy >=2.2.1
     - libpysal
-    - spenc
-    - cenpy >=1.0
     - geopandas
     - matplotlib-base
     - scikit-learn
     - seaborn
     - pyarrow >=0.14.1
-    - appdirs
-    - dash
-    - dash-bootstrap-components
-    - palettable
-    - pip
     - tqdm
     - quilt3 >=3.1.11
     - xlrd
+    - appdirs
     - mapclassify
     - region >=0.2.0
```

Comment:
Is `region` still a dependency?

Reply:
no, missed that one. Feel free to rm and commit directly here